.gitmodules

```diff
@@ -49,3 +49,6 @@
 	shallow = true
 	path = External/robin-map
 	url = https://github.com/Tessil/robin-map.git
+[submodule "External/fex-assorted-tests-bins"]
```

Comment:
Because they also include the binaries, and we don't want to include binaries in the main repo, similar to how posixtests, gcc tests and gvisor tests are linked.

Reply:
Why do the binaries need to be included? The tests you listed are huge (or have heavy dependencies) so it's reasonable not to compile them as part of fex. The SMC tests are just a couple of source files with no dependencies, so it's fine to compile them as part of fex. That has the advantage of making future modifications so much easier (no need to update bins, no need to synch yet another submodule, ...).

Reply:
Three reasons.

One is that we handle all non-assembly unit tests this way right now this way.

Another is that getting them to compile as part of FEX build is non trivial, and an extra pain for non-ubuntu distributions. We've run into this with the thunklibs, not sure to what degree it is resolved.

And, finally, those are not meant to be unit-tests, at least not yet. I mostly wrote them to explore linux/fex behavior, and they rely on undefined posix/linux semantics, don't do proper error testing, so they won't make good tests.

I often develop quick test programs while working on features, as as per the discussions we had a couple of weeks back I made the repo to put them in, so we don't lose them and have to re-do them over and over.

Reply:
(Though, a valid discussion point is if we want them as a submodule. I don't know. It's not very practical, and the repo has already vastly drifted over the commit pointed to here)